dashboard/test/controllers/pd/workshop_daily_survey_controller_test.rb

```diff
@@ -105,8 +105,8 @@ class WorkshopDailySurveyControllerTest < ActionDispatch::IntegrationTest
         user: @enrolled_summer_teacher
 
       sign_in @enrolled_summer_teacher
-      get '/pd/workshop_survey/foorm/day/0'
-      assert_redirected_to action: 'thanks'
+      get '/pd/workshop_pre_survey'
+      assert_thanks
     end
 
     test 'pre-workshop survey displays embedded JotForm when enrolled' do
```

Comment:
Rename?

Reply:
the Jotform test? This test still refers to the Jotform link

Reply:
Oops, github hidden unchanged lines confused me. Ignore!